### PR TITLE
Scanning for the Electronic Warfare System

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug fix**

This PR addresses the bug/feature described in issue none

## Summary
@Zitchas this adds the different types of scan power to the Electronic Warfare System, which I think you forgot in #70.

## Screenshots
None.

## Usage examples
N/A

## Testing Done
None.

## Save File
This save file can be used to test these changes:
None.

## Artwork Checklist
N/A

## Wiki Update
N/A

## Performance Impact
N/A

### CODE REVIEW

File: data/sheragi/sheragi outfits.txt [hunk 1]
```diff
@@ -150,7 +150,14 @@ outfit "Electronic Warfare System"
 	"outfit space" -6
 	"unique" 1
 	"radar jamming" 10
-	"tactical scan power" 25
+	"crew scan power" 20
+	"energy scan power" 40
+	"fuel scan power" 30
+	"maneuver scan power" 25
+	"range finder power" 1
+	"strategic scan power" 20
```
Comment (Zitchas) [inline data/sheragi/sheragi outfits.txt]:
```suggestion
```

Just a note here: "Strategic Scan" is, like "Tactical Scan" actually a conglomerate of the various movement related information types. So having it here alongside them doubles up on things.

**Comment by Zitchas** [comment]
Thank you, yes, I did forget about that one. Good catch!

File: data/sheragi/sheragi outfits.txt (hunk 1 above)
Comment (TheGiraffe3) [reply data/sheragi/sheragi outfits.txt]: Wasn't aware, thanks.

**Comment by Zitchas** [comment]
@Karirawri 

If you are around and can provide some input; what would the strengths and weaknesses of the Sheragi sensor system be?

**Comment by Zitchas** [comment]
I'm not sure if Karirawri or Warlord Mike (DM'd on Discord) are around to provide insight into the Sheragi strengths and weaknesses. I'll take a stab at some stats sometime in the next day or two and we can move forward on this. If they arrive after we've moved forward we can make subsequent PR's to take their input into account.

File: data/sheragi/sheragi outfits.txt [hunk 2]
```diff
@@ -150,7 +150,13 @@ outfit "Electronic Warfare System"
 	"outfit space" -6
 	"unique" 1
 	"radar jamming" 10
-	"tactical scan power" 25
+	"crew scan power" 20
+	"energy scan power" 40
+	"fuel scan power" 30
```
Comment (Zitchas) [inline data/sheragi/sheragi outfits.txt]:
```suggestion
```

File: data/sheragi/sheragi outfits.txt [hunk 3]
```diff
@@ -150,7 +150,13 @@ outfit "Electronic Warfare System"
...
+	"maneuver scan power" 25
```
Comment (Zitchas) [inline data/sheragi/sheragi outfits.txt]:
```suggestion
```

**Review by Zitchas — CHANGES_REQUESTED** [review]
I consulted with Warlord Mike (of the Sheragi Saga, so one of the experts on the Sheragi in my books), and he suggested crew and weapon ranges being the best fit. 

I'd say leave rangefinding and thermal in as well. I've suggested the relevant changes.

File: data/sheragi/sheragi outfits.txt (hunk 2 above)
Comment (TheGiraffe3) [reply data/sheragi/sheragi outfits.txt]: For the energy though, it does say it's an electronic system...

**Comment by TheGiraffe3** [comment]
I just liked the idea of the EWS being able to do everything, but I suppose then you have balance issues.

**Comment by Zitchas** [comment]
So do I. Especially since we don't get a complete ecosystem for the Sheragi. It also makes sense that they'd have some form of info for most of these things, just not at the same level.c

Likewise, the Sheragi are theoretically supposed to be a bit lower tech, overall, than humanity. Not universally so, but overall. Sensors are a bit of a strong point in the way they were initially established, though, so that's reasonable to keep.

And as a side thing, I find that *all* the sensor suites are on the short side and should be a bit farther. 

Hmm... Idea: 
- Bump the current passive sensor values (let's say they are the "core" elements up by 5.
- Add the velocity option at the same range as the weapon scan (identifying how fast something is going is very basic analysis, after all)
- Add energy and fuel at a value of 20, so quite close range. 

This will give a bit of a cascading info effect, which I personally find interesting/neat and make it stand out as something unique.

**Comment by TheGiraffe3** [comment]
Passive sensor values I assume mean the ones currently in?

**Comment by Zitchas** [comment]
Yes, like that.

I suppose I should clarify:

When I refer to "active scanners", I mean the scans that the player has to issue a command to use. Namely, the cargo and outfit scanning.

When I refer to "passive scanners", I mean the things that provide information passively with no more than having to have a target (and not even that, for asteroid scanners).

Anyway, those values look good. I'll merge it once the checks are done.

**Review by Zitchas — APPROVED** [review]

**Comment by Zitchas** [comment]
Thanks for spotting and working on this!

**Comment by TheGiraffe3** [comment]
No problem!

**Pull request merged**